packages/aggregate-api/src/aggregate/offer.js

```diff
@@ -1,11 +1,13 @@
 import * as Server from '@ucanto/server'
 import { CBOR } from '@ucanto/core'
+import { Node, Aggregate as AggregateBuilder } from '@web3-storage/data-segment'
 import * as Aggregate from '@web3-storage/capabilities/aggregate'
 import * as Offer from '@web3-storage/capabilities/offer'
 import * as API from '../types.js'
 
 export const MIN_SIZE = 1 + 127 * (1 << 27)
-export const MAX_SIZE = 127 * (1 << 28)
+// export const MAX_SIZE = 127 * (1 << 28)
+export const MAX_SIZE = AggregateBuilder.DEFAULT_DEAL_SIZE
```

Comment:
@Gozala interesting that these values are different than ones previously obtained with riba. Clues?